.github/workflows/test_suite_ubuntu.yml

```
@@ -62,24 +62,44 @@ jobs:
           . ftorch/bin/activate
           pip install torch torchvision --index-url https://download.pytorch.org/whl/cpu
 
+      # MPI is required by pFUnit
+      - name: Install an MPI distribution
+        run: |
+          sudo apt update
+          sudo apt install mpich
+
+      - name: Install pFUnit
+        run: |
+          export FC=/usr/bin/gfortran
+          export MPIF90=/usr/bin/mpif90
+          # TODO: Avoid version pinning (needed because version appears in install path)
+          git clone -b v4.10.0 https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
+          mkdir pFUnit/build
+          cd pFUnit/build
+          cmake -DMPI=YES ..
+          make install
+
       - name: Build FTorch
         run: |
           . ftorch/bin/activate
           VN=$(python -c "import sys; print('.'.join(sys.version.split('.')[:2]))")
           export Torch_DIR=${VIRTUAL_ENV}/lib/python${VN}/site-packages
           export BUILD_DIR=$(pwd)/src/build
+          # TODO: How can we avoid the pFUnit version ending up in the install path?
+          export PFUNIT_DIR=$(pwd)/pFUnit/build/installed/PFUNIT-4.10
           mkdir ${BUILD_DIR}
           cd ${BUILD_DIR}
           cmake .. \
             -DPython_EXECUTABLE="$(which python)" \
             -DCMAKE_BUILD_TYPE=Release \
             -DCMAKE_INSTALL_PREFIX=${BUILD_DIR} \
             -DCMAKE_BUILD_TESTS=TRUE \
-            -DCMAKE_Fortran_FLAGS="-std=${{matrix.std}}"
+            -DCMAKE_PREFIX_PATH="${PFUNIT_DIR}:${Torch_DIR}" \
```

I am not sure how this works. I think it should be
```suggestion
            -DCMAKE_PREFIX_PATH="${PFUNIT_DIR};${Torch_DIR}" \
```

See [here](https://cmake.org/cmake/help/v3.0/command/list.html) for info on CMake lists.

On my machine it won't run unless I make it a semicolon.

---

Huh, I wasn't aware of that. Thanks!